src/schema/ghga.yaml

```diff
@@ -196,8 +209,17 @@ classes:
       type:
         description: >-
           The type of Study. For example, "Cancer Genomics", "Epigenetics", "Exome Sequencing".
-        # TODO: Enum
+        range: study_type_enum
+        required: true
+      short name:
```

Comment:
I am not sure if short name is really helpful. For whom is the short name? The data submitter? data approver? End user?